src/test/kotlin/no/nav/familie/baks/mottak/søknad/JournalføringTest.kt

```diff
@@ -40,7 +48,7 @@ class JournalføringTest(
     fun `journalPostId blir lagt på dbSøknaden`() {
         val dbSøknadFraDB = søknadService.lagreDBSøknad(dbSøknad)
         assertEquals(null, dbSøknadFraDB.journalpostId)
-        journalføringService.journalførSøknad(dbSøknadFraDB, testPDF)
+        journalføringService.journalførBarnetrygdSøknad(dbSøknadFraDB, testPDF)
```

Comment:
Litt vanskelig å få helt oversikt over PR'en, men har vi disse testene for kontantstøtte også? Eller blir det litt for duplikat?